modules/3d-tiles/src/lib/parsers/helpers/parse-3d-tile-gltf-view.js

```diff
@@ -52,7 +52,7 @@ export function parse3DTileGLTFViewSync(tile, arrayBuffer, byteOffset) {
 export async function extractGLTF(tile, gltfFormat, options, context) {
   extractGLTFBufferOrURL(tile, gltfFormat, options);
 
-  if (options['3d-tiles'] && options['3d-tiles'].loadGLTF) {
+  if (options.loadGLTF) {
```

Comment:
This is not correct. loaders.gl 2.0 supports nested options objects for each loader. See docs at https://github.com/uber-web/loaders.gl/blob/master/modules/3d-tiles/docs/api-reference/tile-3d-loader.md

Reply:
https://github.com/uber-web/loaders.gl/blob/a2b7ee89cff5db53114f18dba58e911b6708ca5a/modules/3d-tiles/src/lib/parsers/parse-3d-tile.js#L14
Here we did one extraction, currently, the batched models all failed because of this check here.